Ghidra/Processors/PowerPC/data/languages/altivec.sinc

```diff
@@ -246,6 +248,16 @@ define pcodeop vectorUnpackLowSignedHalfWord;
 	vrD = loadVectorIndexedLRU(A,B);
 }
 
+:lvlx vrD,A,B      is OP=31 & vrD & A & B & XOP_1_10=519 & Rc=0
```

Comment:
I think both of these `A` should be `RA_OR_ZERO`.

binutils at least appears to have them so
`{"lvlx",        X(31,519),      X_MASK,      CELL,      0,              {VD, RA0, RB}},`
`{"lvrx",        X(31,551),      X_MASK,      CELL,      0,              {VD, RA0, RB}},`

Reply:
you are correct, https://arcb.csc.ncsu.edu/~mueller/cluster/ps3/SDK3.0/docs/arch/vector_simd_pem_v_2.07c_26Oct2006_cell.pdf

There appears to be more instructions in this file that should be `RA_OR_ZERO`, will fix in a followup PR